libraries/AP_HAL_ChibiOS/hwdef/common/user_settings.h

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
use wolfssl subdir I think, and a small README.md in there

Reply:
done